cranelift/fuzzgen/src/function_generator.rs

```diff
@@ -691,7 +691,7 @@ where
         let sig = self.generate_signature()?;
 
         let mut fn_builder_ctx = FunctionBuilderContext::new();
-        let mut func = Function::with_name_signature(ExternalName::user(0, 0), sig.clone());
+        let mut func = Function::with_name_signature(ExternalName::user(0, 1), sig.clone());
```

Comment:
What is this change for?

Reply:
The name `user(0, 0)` was the trampoline IIRC and the JIT was complaining about it. I'm not too sure, but I can go check.